plugins/brimcap/brimcap-cli.ts

```diff
@@ -63,10 +63,13 @@ export default class BrimcapCLI {
 
   private exec(subCommand: string, opts: searchOptions) {
     const commandWithArgs = [
-      this.binPath,
+      `"${this.binPath}"`,
       subCommand,
       ...flatMap(
-        Object.entries(opts).map(([k, v]) => [`-${OPTION_NAME_MAP[k] || k}`, v])
+        Object.entries(opts).map(([k, v]) => [
+          `-${OPTION_NAME_MAP[k] || k}`,
+          `"${v}"`
+        ])
```

Comment:
This can go sideways if `this.binPath` or any `opts` value contains a quotation mark (very unlikely but possible). Make it bulletproof by switching from `execSync` to `spawnSync`!